recipes-ofdpa/ofdpa/ofdpa-platform/platform/arm-accton-as4610-54-r0/rc.soc

```diff
@@ -0,0 +1,131 @@
+linkscan off
+phy raw 0x9 0x1f 0xffd0
+phy raw 0x9 0x1e 0x0
+phy raw 0x9 0x1f 0x8060
+phy raw 0x9 0x17 0x03e2
+phy raw 0x9 0x1f 0xffd0
+phy raw 0x9 0x1e 0x4
+phy raw 0x9 0x1f 0x8060
+phy raw 0x9 0x17 0x03e2
+phy raw 0x13 0x1f 0xffd0
+phy raw 0x13 0x1e 0x0
+phy raw 0x13 0x1f 0x8060
+phy raw 0x13 0x17 0x03e2
+phy raw 0x13 0x1f 0xffd0
+phy raw 0x13 0x1e 0x4
+phy raw 0x13 0x1f 0x8060
+phy raw 0x13 0x17 0x03e2
+phy raw 0x1d 0x1f 0xffd0
+phy raw 0x1d 0x1e 0x0
+phy raw 0x1d 0x1f 0x8060
+phy raw 0x1d 0x17 0x03e2
+phy raw 0x1d 0x1f 0xffd0
+phy raw 0x1d 0x1e 0x4
+phy raw 0x1d 0x1f 0x8060
+phy raw 0x1d 0x17 0x03e2
+phy raw 0x29 0x1f 0xffd0
+phy raw 0x29 0x1e 0x0
+phy raw 0x29 0x1f 0x8060
+phy raw 0x29 0x17 0x01e2
+phy raw 0x29 0x1f 0xffd0
+phy raw 0x29 0x1e 0x4
+phy raw 0x29 0x1f 0x8060
+phy raw 0x29 0x17 0x01e2
+phy raw 0x33 0x1f 0xffd0
+phy raw 0x33 0x1e 0x0
+phy raw 0x33 0x1f 0x8060
+phy raw 0x33 0x17 0x01e2
+phy raw 0x33 0x1f 0xffd0
+phy raw 0x33 0x1e 0x4
+phy raw 0x33 0x1f 0x8060
+phy raw 0x33 0x17 0x01e2
+phy raw 0x3d 0x1f 0xffd0
+phy raw 0x3d 0x1e 0x0
+phy raw 0x3d 0x1f 0x8060
+phy raw 0x3d 0x17 0x01e2
+phy raw 0x3d 0x1f 0xffd0
+phy raw 0x3d 0x1e 0x4
+phy raw 0x3d 0x1f 0x8060
+phy raw 0x3d 0x17 0x01e2
+phy int ge0 0x1f 0xffd0 
+phy int ge0 0x1e 0x0       
```

Comment:
There's gratuitous trailing whitespace here and in other files.

Reply:
It's how it is in the source files, so I left it as is to make the comparison easier.